docs/building_pipelines.rst

```diff
@@ -293,3 +294,423 @@ Warnings raised will also be logged to the error log but will not cause data to
+             "reading": {
+               "gearbox": 27.93525,
+               "motor": 29.6282,
+               "headstock": 25.51,
+               "tailstock": 20.52,
+               "tool": 24.0905
+             }
+           }
+         ]
+       },
+       [
+         {
+           "name": "CurrentOnly",
+           "readings": [
+             {
+               "asset_code": "latheIR",
+               "user_ts": "2025-03-31 16:23:10.008024+00:00",
+               "ts": "2025-03-31 16:23:10.008024+00:00",
+               "reading": {
+                 "gearbox": 27.93525,
+                 "motor": 29.6282,
+                 "headstock": 25.51,
+                 "tailstock": 20.52,
+                 "tool": 24.0905
+               }
+             }
+           ]
+         },
+         {
+           "name": "RMS",
+           "readings": [
+             {
+               "asset_code": "latheCurrent",
+               "user_ts": "2025-03-31 16:23:10.008018+00:00",
+               "ts": "2025-03-31 16:23:10.008018+00:00",
+               "reading": {
+                 "current": 767
+               }
+             }
+           ]
+         },
+         {
+           "name": "Writer",
+           "readings": [
+             {
+               "asset_code": "latheCurrent RMS",
+               "user_ts": "2025-03-31 16:23:02.803881+00:00",
+               "ts": "2025-03-31 16:23:02.803881+00:00",
+               "reading": {
+                 "current": 773.9413414465,
+                 "currentpeak": 47,
+                 "currentcrest": 0.0607281166
+               }
+             }
+           ]
+         }
+       ],
+       {
+         "name": "NonCurrent",
+         "readings": [
+           {
+             "asset_code": "latheIR",
+             "user_ts": "2025-03-31 16:23:10.008024+00:00",
+             "ts": "2025-03-31 16:23:10.008024+00:00",
+             "reading": {
+               "gearbox": 27.93525,
+               "motor": 29.6282,
+               "headstock": 25.51,
+               "tailstock": 20.52,
+               "tool": 24.0905
+             }
+           }
+         ]
+       },
+       {
+         "name": "Writer",
+         "readings": [
+           {
+             "asset_code": "latheIR",
+             "user_ts": "2025-03-31 16:23:10.008024+00:00",
+             "ts": "2025-03-31 16:23:10.008024+00:00",
+             "reading": {
+               "gearbox": 27.93525,
+               "motor": 29.6282,
+               "headstock": 25.51,
+               "tailstock": 20.52,
+               "tool": 24.0905
+             }
+           }
+         ]
+       }
+     ]
+   }
+   Debug: Lathe$
+
+Data is displayed as a JSON document, each object in the data represents a node in the pipeline. Each pipeline is enclosed in a JSON array. The *name* property of the object is the name of the filter into which the data is about to be passed. There are two reserved names; *Branch* and *Writer*.
```

Comment:
- Start a new sentence at "Each object in the data represents a node..."
- There are two reserved names; -> There are two reserved names: